### DISCUSSION TITLE
Recommended setup method

### DISCUSSION THREAD

**Discussion by schiessle** [mastodon/mastodon#22640] [Q&A] [answered]
Hi all,

I consider settings up my own Mastodon instance and wonder what's the best way to do so.

On docs.joinmastodon.org I only find documentation how to install Mastodon from source but on the internet I can also find quite some information  about a Docker setup.

What does most people use? Is there a preference / recommended way to install Mastodon in production? Also with respect to long term maintainability, updates, support, etc

Thanks!
Björn

**Comment by nightpool** [comment, marked as answer]
We do not recommend people use Docker unless they already have substantial prior experience deploying containerized applications in a production environment.

**Reply by schiessle** [reply]
Thanks for the quick feedback. I don't have a lot of experience with Docker. That's why I wanted to make sure to chose the deployment method where most people are familiar with and where most people can help in case of issues. So I will continue with a "from source" deployment.
